sql/core/src/main/scala/org/apache/spark/sql/execution/SparkStrategies.scala

```diff
@@ -571,9 +571,13 @@ abstract class SparkStrategies extends QueryPlanner[SparkPlan] {
           resultExpressions,
           planLater(child)))
 
-      case PhysicalAggregation(_, _, _, _) =>
+      case PhysicalAggregation(_, aggExpressions, _, _) =>
+        var groupAggPandasUDFNames = Set[String]()
```

Comment:
Also, I think we should add a default string in case `groupAggPandasUDFNames` is empty to make the code future-proof.